app/services/feeds/import.rb

```diff
@@ -0,0 +1,192 @@
+# TODO: [rhymes]
+# => add Feeds::ImportUser to fetch a single user
+# => add Feeds::ValidateFeedUrl to validate a single feed URL
+module Feeds
+  class Import
+    def self.call(silent: true)
+      # TODO: remove this eventually
+      if silent && Rails.env.development?
+        ActiveRecord::Base.logger = nil
+        Rails.logger.level = :info
+        Rails.configuration.log_level = :info
+      end
+
+      new.call
+    end
+
+    # TODO: add `users` param
+    def initialize
+      @users = User.where.not(feed_url: [nil, ""])
+
+      # NOTE: should these be configurable? Currently they are the result of empiric
+      # tests trying to find a balance between memory occupation and speed
+      @users_batch_size = 50
+      @num_fetchers = 8
+      @num_parsers = 4
+    end
+
+    def call
+      total_articles_count = 0
+
+      users.in_batches(of: users_batch_size) do |batch_of_users|
+        feeds_per_user_id = fetch_feeds(batch_of_users)
+        Rails.logger.info("feeds_per_user_id.length: #{feeds_per_user_id.length}")
```

Comment:
Is this data we will want to gather from production? If so, our log level is ERROR so we will have to make these error logs if we want to see them, which I am fine with for a few test runs.

Reply:
Nope, these should go, forgot to remove them